makefile

```diff
@@ -1,3 +1,6 @@
+# Note: this makefile expects you to have run `npm install` and `npm run build`
+# first before using it.
+
 _ := $(shell rm -rf build && npm run build)
```

Comment:
```suggestion
_ := $(shell rm -rf build && npm install && npm run build)
```

Wouldn't this work to run all of the necessary commands beforehand?

Reply:
I mean, I'm kinda wary about adding further stuff to that command...

Reply:
> I mean, I'm kinda wary about adding further stuff to that command...

...but yeah, I agree that we should check with @kheina